src/workspace.js

```diff
@@ -1744,13 +1746,18 @@ module.exports = class Workspace extends Model {
     }
   }
 
-  // Restore to the window's original editor font size.
+  // Restore to the window's default editor font size.
   resetFontSize() {
-    if (this.originalFontSize) {
-      this.config.set('editor.fontSize', this.originalFontSize);
+    if (this.defaultFontSize) {
+      this.config.set('editor.fontSize', this.defaultFontSize);
```

Comment:
`this.originalFontSize` was only needed to reset the font size to what it was when the window was opened. I don't think `this.defaultFontSize` is needed since it will always be the same as `atom.config.get('editor.defaultFontSize')`.
```suggestion
    this.config.set('editor.fontSize', this.config.get('editor.defaultFontSize'));
```

Reply:
Removed the subscription as well. 👍